### PR TITLE
Remove Hadoop dependency

### PR DESCRIPTION
**Pull request by afoerster**

### CODE REVIEW

**Comment by afoerster** [comment]
@kjmccarthy I fixed the logging issue by adding log4j2. Any conflict we might have had with log4j1 (guessing this is what was happening) was removed with the Hadoop dependency.

**Comment by afoerster** [comment]
3 tests are failing now in `DefaultProvisioningServiceSpec.scala`. I'm not sure what changed but I'm not going to dig into it now, the whole provisioning thing is difficult to reason about and needs to be refactored. That will be another task

File: integration-test/src/test/scala/io/phdata/clients/SentryClientIntegrationSpec.scala [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+import java.util.UUID
+
+import cats.effect.IO
+import io.phdata.itest.fixtures.{HiveTest, IntegrationTest, _}
+import io.phdata.models.{Manager, ReadOnly, ReadWrite}
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+import cats.implicits._
+
+class SentryClientIntegrationSpec extends FlatSpec with Matchers with HiveTest with IntegrationTest with BeforeAndAfterAll{
```
Comment (kjmccarthy) [inline integration-test/src/test/scala/io/phdata/clients/SentryClientIntegrationSpec.scala]: Was this supposed to be added back in? I changed the name in the last PR

File: common/src/main/scala/io/phdata/services/CDHClusterService.scala [hunk 2]
```diff
@@ -109,7 +107,7 @@ class CDHClusterService[F[_]: ConcurrentEffect: Clock](
           "hive",
           hive,
           hosts,
-          Map("thrift" -> (hadoopConfiguration.get("hive.server2.thrift.port", "10000").toInt, hiveServer2Roles))
+          Map()
```
Comment (kjmccarthy) [inline common/src/main/scala/io/phdata/services/CDHClusterService.scala:110]: What does this empty map do?

File: project/Dependencies.scala [hunk 3]
```diff
@@ -106,18 +89,16 @@ object Dependencies {
...
+  val scalaLoggingVersion = "3.8.0"
+  val log4j2Version = "2.14.0"
 
   val logging = Seq(
-    "com.typesafe.scala-logging" %% "scala-logging" % loggingVersion
+    "com.typesafe.scala-logging" %% "scala-logging" % scalaLoggingVersion,
+    "org.apache.logging.log4j" % "log4j-api" % log4j2Version
   )
 
+  Seq()
```
Comment (kjmccarthy) [inline project/Dependencies.scala]: ?

File: provisioning/src/main/scala/io/phdata/provisioning/LocationGrantProvisioning.scala [hunk 4]
```diff
@@ -12,7 +12,7 @@ object LocationGrantProvisioning {
   implicit val show: Show[LocationGrant] =
     Show.show(g => s"""role "${g.roleName}" access to "${g.location}"""")
 
-  implicit object LocationGrantProvisioningTask extends ProvisioningTask[LocationGrant] {
+  object LocationGrantProvisioningTask extends ProvisioningTask[LocationGrant] {
```
Comment (kjmccarthy) [inline provisioning/src/main/scala/io/phdata/provisioning/LocationGrantProvisioning.scala:15]: Nice

File: integration-test/src/test/scala/io/phdata/clients/SentryClientIntegrationSpec.scala (hunk 1 above)
Comment (kjmccarthy) [reply integration-test/src/test/scala/io/phdata/clients/SentryClientIntegrationSpec.scala]: Looks like it's outdated.

File: common/src/main/scala/io/phdata/services/CDHClusterService.scala (hunk 2 above)
Comment (afoerster) [reply common/src/main/scala/io/phdata/services/CDHClusterService.scala:110]: Needed there just for the argument now, it was originally to set a default thrift server port, I think it was pulled from the Hadoop conf. Don't need it now.

File: project/Dependencies.scala (hunk 3 above)
Comment (afoerster) [reply project/Dependencies.scala]: oye

**Pull request merged**